src/main/kotlin/org/rust/lang/core/psi/BuiltinAttributes.kt

```diff
@@ -5,188 +5,94 @@
 
 package org.rust.lang.core.psi
 
+import com.fasterxml.jackson.module.kotlin.jacksonObjectMapper
+import com.fasterxml.jackson.module.kotlin.readValue
+import com.intellij.openapi.diagnostic.Logger
+import com.intellij.openapi.diagnostic.logger
+import java.io.IOException
+
 
 // Bump `RsFileStub.Type.STUB_VERSION` if modified
 // https://github.com/rust-lang/rust/blob/7a8636c843bd24038fe1d1f69b4a8e4b0ea55d4e/compiler/rustc_feature/src/builtin_attrs.rs#L266
```

Comment:
It seems this comment is not relevant anymore. At least, with concrete commit hash